README.md

```diff
@@ -5,22 +5,36 @@
 No matter which environment you will need to setup your Open AI API Key.
 
 Create a .env file in the project root directory with the following:
-OPENAI_API_KEY=
+```
+OPENAI_API_KEY=<paste your own api key here>
+STREAM_KEY_NAME=hello
```

Comment:
Not yet fully implemented (WIP) - All RTMP streams currently go into the `/live_audio` directory, potentially mixing individual livestreams.

Reply:
Our current best option might be to use configure Nginx-rtmp to **only** accept streams with the set `STREAM_KEY_NAME`, effectively limiting the current setup to one stream only, with a somewhat private key name.